src/bridge/AbsBridge.sol

```diff
@@ -55,8 +59,25 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
 
     uint256 public override sequencerReportedSubMessageCount;
 
+    uint256 public totalDelayedMessagesRead;
+
     address internal constant EMPTY_ACTIVEOUTBOX = address(type(uint160).max);
 
+    event SequencerBatchDelivered(
```

Comment:
we should document that the signature of this event will change due to the new packing of `TimeBounds`

Reply:
Added desc on PR